jbpm-wb-process-runtime/jbpm-wb-process-runtime-api/src/main/java/org/jbpm/workbench/pr/model/ProcessDefinitionDataSetProviderType.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.pr.model;
+
+import org.dashbuilder.dataprovider.BeanProviderType;
+import org.dashbuilder.dataset.json.DataSetDefJSONMarshallerExt;
+import org.jboss.errai.common.client.api.annotations.Portable;
+
+@Portable
+public class ProcessDefinitionDataSetProviderType extends BeanProviderType {
+
+    @Override
+    public String getName() {
+        return "PROCESS DEFINTIONS";
```

Comment:
please move to static field.
```suggestion
        return "PROCESS DEFINITIONS";
```